### PR TITLE
Add new tally avatar

### LINKED ISSUE #1184: Account - Old Tally Logo
image

### PR DESCRIPTION
**Pull request by 0xDaedalus**
Closes #1184 

Adds the correct tally avatar to the Accounts list.

image

### CODE REVIEW

File: ui/components/AccountsNotificationPanel/AccountsNotificationPanelAccounts.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const walletTypeDetails: { [key in AccountType]: WalletTypeInfo } = {
   },
   [AccountType.Internal]: {
     title: "Tally Ho",
-    icon: "./images/tally_reward@2x.png", // FIXME: Icon is cut off - we should get a better one
+    icon: "./images/tally_avatar.svg", // FIXME: Icon is cut off - we should get a better one
```
Comment (Shadowfiend) [inline ui/components/AccountsNotificationPanel/AccountsNotificationPanelAccounts.tsx:42]: Is the icon still cut off?

Comment (0xDaedalus) [reply ui/components/AccountsNotificationPanel/AccountsNotificationPanelAccounts.tsx:42]: It is not, good catch

**Review by VladUXUI — COMMENTED** [review]
Looks good from the screenshot!

**Review by greg-nagy — APPROVED** [review]
let's 🚢

**Pull request merged**